site/src/pages/GroupsPage/GroupPage.tsx

```diff
@@ -125,9 +85,9 @@ export const GroupPage: React.FC = () => {
                     <Button startIcon={<SettingsOutlined />}>Settings</Button>
                   </Link>
                   <Button
-                    disabled={group?.id === group?.organization_id}
+                    disabled={groupData?.id === groupData?.organization_id}
```

Comment:
This comes down to a UX thing, but does it make sense to outright disable the button?
https://adamsilver.io/blog/the-problem-with-disabled-buttons-and-what-to-do-instead/

Reply:
I think it is ok for now but I agree that maybe there are better ways to make that.

Reply:
Will do